securedrop/journalist_templates/edit_account.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{% block body %}
+<h2>Edit your account</h2>
+
+<form method="post">
+  <input name="csrf_token" type="hidden" value="{{ csrf_token() }}"/>
+  <p>
+    <label for="password">Change password</label>
+    <input name="password" id="password" type="password" />
+    <em class="light">Leave blank for no change</em>
```

Comment:
This is no longer necessary because the only thing you're allowing them to change is their password. This *was* necessary (albeit arguably not a great UX) on the admin's pages because they could also change the user's username and toggle `is_admin`, so there needed to be a way to signal "I'm just changing the username and/or the is_admin toggle, and I don't want to reset their password to an empty password at the same time".